src/MenuItem/StaticItem.php

```diff
@@ -2,23 +2,30 @@
 
 namespace PhpSchool\CliMenu\MenuItem;
 
-use Assert\Assertion;
 use PhpSchool\CliMenu\MenuStyle;
 use PhpSchool\CliMenu\Util\StringUtil;
+use PhpSchool\CliMenu\Style\SelectableStyle;
 
 /**
  * @author Michael Woodward <mikeymike.mw@gmail.com>
  */
-class StaticItem implements MenuItemInterface
+class StaticItem implements MenuItemInterface, SelectableStyleInterface
```

Comment:
This is odd - it's not a selectable item. Can we drop the interface please, I'm going to try to think of a better way to do the style propagation.

Reply:
It's not selectable, but shares styles with selectable item, and splititem accepts static and calls the style methods.

Reply:
Yeah the name is not right so I'd rather drop it for now and think of a better way to do it before the next release